numba_dppy/compiler.py

```diff
@@ -154,7 +159,7 @@ def compile_kernel(sycl_queue, pyfunc, args, access_types, debug=None):
         # We expect the sycl_queue to be provided when this function is called
         raise ValueError("SYCL queue is required for compiling a kernel")
 
-    cres = compile_with_dppy(pyfunc, None, args, debug=debug)
+    cres = compile_with_dppy(pyfunc, None, args, True, debug=debug)
```

Comment:
Use kwargs (return_type=None, is_kernel=True) as that makes reading the code much easier and understanding what each argument does.

Reply:
I do not want to use kwargs here for the following reasons:
1. Keyword arguments are optional unless we make all arguments keyword arguments and use techniques described here (https://www.vegardstikbakke.com/python-keyword-only/). Please let me know if this is a wrong assumption.
2. I do not want the function to assume a default value as I want future developers to explicitly tell this function whether this is a kernel or not. In case keyword arguments are optional, future developers might not provide it and the default value will be assumed.

Making the code readable is the only positive aspect I see if my understanding is correct. Please correct me if I am under wrong assumptions.